go/private/rules/transition.bzl

```diff
@@ -319,8 +320,8 @@ forwards Go providers and DefaultInfo.
 
 This is used to work around a problem with building tools: Go tools should be
 built with 'cfg = "exec"' so they work on the execution platform, but we also
-need to apply go_tool_transition so that e.g. a tool isn't built as a shared
-library with race instrumentation. This acts as an intermediate rule that allows
+need to apply go_tool_transition so that, for example, a tool isn't built as a shared
```

Comment:
This mentions a symbol that itself isn't public.

Reply:
Good point, tried to update it to be a bit more clear.